pallets/rmrk-core/src/functions.rs

```diff
@@ -313,9 +319,8 @@ where
 
 		for _ in Resources::<T>::drain_prefix((collection_id, nft_id)) {}
 
-		let kids = Children::<T>::take((collection_id, nft_id));
-		for (child_collection_id, child_nft_id) in kids {
-			// Remove child from Children StorageMap
+		let c = Children::<T>::iter_prefix((collection_id, nft_id,)).count();
```

Comment:
What is this used for?

Reply:
nothing.  removed. fab8c5e569c187e450c20120952bf6794bf8d169